### PR TITLE
Updates auth related imports for graphql-server

### PR DESCRIPTION
**Pull request by dthyresson**
First:

Several auth-related templates using in such setup references errors imported from `redwoodjs/api`.

These errors now live in `redwoodjs/graphql-server` and as such their imports need updating.

Second:

The general `getCurrentUser` in the auth.js shipped could always return and object and set as `currentIser` in the context.

The change aims to reduce that risk if just passing back a decoded token -- without roles.

Open to code improvements here, just wanted to get the basic idea out there.

### CODE REVIEW

File: packages/cli/src/commands/setup/auth/templates/auth.js.template [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { AuthenticationError, ForbiddenError, parseJWT } from '@redwoodjs/api'
+import { AuthenticationError, ForbiddenError, parseJWT } from '@redwoodjs/graphql-server'
```
Comment (dac09) [inline packages/cli/src/commands/setup/auth/templates/auth.js.template:1]: parseJWT is still in @redwoodjs/api I believe

Comment (dthyresson) [reply packages/cli/src/commands/setup/auth/templates/auth.js.template:1]: Whoops. You are correct. Thanks!

File: packages/cli/src/commands/setup/auth/templates/auth.js.template [hunk 2]
```diff
@@ -17,7 +17,17 @@ export const getCurrentUser = async (
   { _token, _type },
   { _event, _context }
 ) => {
-  return { ...decoded, roles: parseJWT({ decoded }).roles }
+  if (decoded) {
```
Comment (dac09) [inline packages/cli/src/commands/setup/auth/templates/auth.js.template:20]: This makes sense, but I would just put the return first 
```js
if(!decoded) {
 return null
}
```

but just personal preference here

**Review by thedavidprice — COMMENTED** [review]
Looks good to me and changes = understood.

We should communicate this in Release Notes, but it will only apply to projects that kept vanilla Auth.js template, correct?

**Pull request merged**